frontend/components/exercise/Checkbox.tsx

```diff
@@ -1,9 +1,25 @@
-import styles from "./Checkbox.module.css";
+import React from "react";
 
-export default function Checkbox() {
+export default function Checkbox({ onClick, className }) {
   return (
-    <div className="flex h-[18px] w-[18px] items-center justify-center">
-      <input type="checkbox" className={`${styles.checkbox} appearance-none`} />
+    <div
+      className={`ml-auto flex h-[16px] w-[16px] items-center justify-center ${className}`}
```

Comment:
double-check width and height stuff esp since with the styling. make sure that the stuff i did in the styles.checkbox aren't nullified if that makes sense?